pkg/cli/connect.go

```diff
@@ -28,10 +28,22 @@ func init() {
 }
 
 var connectCmd = &cobra.Command{
-	Use:   "connect",
+	Use:   "connect [type_of_node:attribute:value] [type_of_node:attribute:value]",
 	Short: "Connect nodes with specific edges",
 	Long: `Usage: qmstrctl connect <that> <this>...
-Connect to Node <that> Node(s) <this>. In case of multiple edges for the specified types you can use --<type>To<type>Edge flag to specify the edge you want.`,
+Connect to Node <that> Node(s) <this>. In case of multiple edges for the specified types you can use --<type>To<type>Edge flag to specify the edge you want.
+
+input: [type_of_node:attribute:value]
+
+type_of_node:
+	- package
+	- file
+attribute:
+	- hash
```

Comment:
So package nodes have hashes? Also every time we change something in the data model we have to adapt this help message. So please generate this text. Have a look at the generateFlags function to get the idea.